.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -52,24 +51,33 @@ body:
     attributes:
       label: "How to reproduce the issue"
       description: |
-        - Steps to reproduce using `nvim -u NORC` or `nvim -u NONE` (try both).
+        - [Prepare](https://github.com/neovim/neovim/blob/master/CONTRIBUTING.md#reporting-problems) a minimal config that reproduces the problem with `nvim -u minimal.lua` and provide it below.
         - For build failures: list the exact steps including CMake flags (if any).
         - For shell-related problems: try `env -i TERM=ansi-256color "$(which nvim)"`.
       placeholder: |
-        nvim -u NONE
-        :edit foo
-        yiwp
+        1. `nvim -u minimal.lua` 
+        2. `:edit foo` 
+        3. `yiwp` 
```

Comment:
this is harder to type for the user, and not really more readable for triagers. and it makes copying the text with a mouse, annoying.

Reply:
It's purely cosmetic for the placeholder?

Reply:
safari bugs (temporary) don't dictate our issue template. it's not purely cosmetic, because this establishes the pattern that many will follow or feel urged to follow.